src/components/MerchSection/MerchData.ts

```diff
@@ -1,51 +1,33 @@
-import BlackHoodie from "../../images/merch/BlackHoodie.jpeg";
-import BlackBeanie from "../../images/merch/BlackBeanie.jpeg";
-import ToteBag from "../../images/merch/ToteBag.jpeg";
-import BlackShortSleeve from "../../images/merch/BlackShortSleeve.jpeg";
-import MousePad1 from "../../images/merch/MousePad1.jpeg";
+import BasicCrewneck from "../../images/merch/BasicCrewneck.jpg";
+import BasicCrewneckWithSleevePrint from "../../images/merch/BasicCrewneckWithSleevePrint.jpg";
 
 export type MerchComponents = {
```

Comment:
Having "components" in the type name is a little misleading, please rename to something like `Merch`, `MerchProperties`, etc.